src/main/java/org/traccar/web/client/view/NavView.java

```diff
@@ -151,33 +140,29 @@ public NavView(SettingsHandler settingsHandler,
         uiBinder.createAndBindUi(this);
 
         User user = ApplicationContext.getInstance().getUser();
-        boolean readOnly = user.getReadOnly();
-        boolean admin = user.getAdmin();
-        boolean manager = user.getManager();
 
-        settingsButton.setVisible(admin || manager);
-        settingsAccount.setVisible(!readOnly);
-        settingsPreferences.setVisible(!readOnly);
+        settingsButton.setVisible(user.hasPermission(UserPermission.SERVER_MANAGEMENT)
+            || user.hasPermission(UserPermission.USER_MANAGEMENT));
+        settingsAccount.setVisible(true);
```

Comment:
Czy tu nie potrzeba USER_MANAGEMENT?

Reply:
Jeżeli ktoś nie ma uprawnienia USER_MANAGEMENT to nie może zarządzać swoim kontem (zmienić imienia, nazwiska, firmy...)?

Reply:
Obecnie tak, ale w takim razie to poprawię.